src/cmd/connect.go

```diff
@@ -14,26 +20,20 @@ var (
 	cliOnly             bool
 
 	connectCmd = &cobra.Command{
-		Use:     "connect {REGISTRY|LOGGING|GIT|connect-name}",
+		Use:     lang.CmdConnect,
 		Aliases: []string{"c"},
-		Short:   "Access services or pods deployed in the cluster.",
-		Long: "Uses a k8s port-forward to connect to resources within the cluster referenced by your kube-context.\n" +
-			"Three default options for this command are <REGISTRY|LOGGING|GIT>. These will connect to the Zarf created resources " +
-			"(assuming they were selected when performing the `zarf init` command).\n\n" +
-			"Packages can provide service manifests that define their own shortcut connection options. These options will be " +
-			"printed to the terminal when the package finishes deploying.\n If you don't remember what connection shortcuts your deployed " +
-			"package offers, you can search your cluster for services that have the 'zarf.dev/connect-name' label. The value of that label is " +
-			"the name you will pass into the 'zarf connect' command. \n\n" +
-			"Even if the packages you deploy don't define their own shortcut connection options, you can use the command flags " +
-			"to connect into specific resources. You can read the command flag descriptions below to get a better idea how to connect " +
-			"to whatever resource you are trying to connect to.",
+		Short:   lang.CmdConnectShort,
+		Long:    lang.CmdConnectLong,
 		Run: func(cmd *cobra.Command, args []string) {
 			var target string
 			if len(args) > 0 {
 				target = args[0]
 			}
 
-			tunnel := k8s.NewTunnel(connectNamespace, connectResourceType, connectResourceName, connectLocalPort, connectRemotePort)
+			tunnel, err := cluster.NewTunnel(connectNamespace, connectResourceType, connectResourceName, connectLocalPort, connectRemotePort)
+			if err != nil {
+				message.Fatal(err, "Unable to create a tunnel into the cluster")
```

Comment:
yes..... 😭

Reply:
As discussed (in person-ish), this PR will set the baseline for lang file capability and we'll slowly go about utilizing the lang file as we complete future work instead of also trying to cram that into this PR.